### ISSUE TITLE
How to add controls to an existing section

### ISSUE THREAD

**Issue by dlzi** [RiseLedger/rise-options#2]
Currently is not possible to add new controls to an existing section, or at least I couldn't figure out how to do it. For example. The above doesn't work:

```
RiseOptions::image('image_logo', 'Upload Your Logo', 'title_tagline' ); 
```

**Comment by RiseLedger** [comment]
Currently you can add the controls only to the sections that was created by RiseOptions::section and not to the already existing sections by wordpress.
Every new control created will be added to the latest section that was register. For example :

RiseOptions::section('Main Options'); // create a new section
RiseOptions::image('image_logo', 'Upload Your Logo'); // create a image type field

RiseOptions::section('Another Section'); // create another section
RiseOptions::image('favicon', 'Upload Your Favicon');
RiseOptions::checkbox('test_checkbox', 'The Checkbox Field', true);

Section "Main Options" will have only image_logo field and section "Another Options" will have favicon and test_checkbox field.

**Comment by dlzi** [comment]
I noticed that. Thanks. 

Do you have any plans to allow controls in already existing sections?

**Comment by RiseLedger** [comment]
Yes, for future updates I think it will be great to have this. Will let you know here when it will be possible. Thanks
